### DISCUSSION TITLE
Dealing with large models and the resulting OWL files

### DISCUSSION THREAD

**Discussion by russell-d-e** [SemanticApplicationDesignLanguage/sadl#780] [Q&A]
I have a large SADL model that I am trying to import into SemTK and I am running into an issue were the OWL file that is created by SADL is 1.2MB and the maximum size that can be loaded by SemTK is 1MB.

I tried splitting the model into three smaller sadl files, however the largest of resulting owl file is larger than the original single OWL file. The model is rather interconnected so I am not sure if would be be able to split the model more without running into a large headache of circular dependencies.

Is there any tips or tricks for limiting the size of the resulting owl model, or is there any tool that are available that would help to split the resulting owl model into smaller files?

**Comment by crapo** [comment]
I'm unaware of any tools, tips, or tricks for splitting up interconnected OWL models. Does SemTK accept files in different formats? A model in N3 or n-triple format might be smaller than one in RDF/XML format. Why is SemTK limited to 1 MB?
